trade-finance-manager-ui/server/controllers/utilisation-reports/find-reports-by-year/index.ts

```diff
@@ -107,24 +93,28 @@ export const getFindReportsByYear = async (req: Request, res: Response) => {
         yearError,
         selectedBank: isNonEmptyString(bankIdQuery) ? bankIdQuery : undefined,
         selectedYear: isNonEmptyString(yearQuery) ? yearQuery : undefined,
-      });
+      };
+
+      return res.render('utilisation-reports/find-utilisation-reports-by-year.njk', viewModel);
     }
 
     const { bankName, year, reports } = await api.getReportSummariesByBankAndYear(userToken, bankIdAsString, yearAsString);
 
     const reportSummaryItemsViewModel = reports.map(getFindReportSummaryItemViewModel);
     const isTfmPaymentReconciliationFeatureEnabled = isTfmPaymentReconciliationFeatureFlagEnabled();
 
-    return renderUtilisationReportsByBankAndYearResults(res, {
+    const viewModel: UtilisationReportsByBankAndYearViewModel = {
       user,
       activePrimaryNavigation: PRIMARY_NAVIGATION_KEYS.UTILISATION_REPORTS,
       bankName,
       year,
       reports: reportSummaryItemsViewModel,
       isTfmPaymentReconciliationFeatureFlagEnabled: isTfmPaymentReconciliationFeatureEnabled,
-    });
+    };
+
+    return res.render('utilisation-reports/utilisation-reports-by-bank-and-year-results.njk', viewModel);
```

Comment:
Similarly can we pull this path into a const at the top of the file and use in lines 76 and 98 and here please

Reply:
done